FBKVOControllerTests/FBKVOTesting.m

```diff
@@ -23,6 +23,17 @@ - (NSString *)debugDescription
 
 @end
 
+BOOL FBKVOTestCircleDeallocatingWasDeallocated = NO;
```

Comment:
Maybe rather than a global variable, have a `BOOL *` property that you set in dealloc? Then each test case can have its own local variable.